package.json

```diff
@@ -12,20 +12,23 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "typecheck": "tsc",
+    "test": "npm run build && npx playwright test"
```

Comment:
@ff6347 not sure about running a build before running the tests. In the workflow it is an own step. Any preference on this? 
I added `npm run build` here because I usually forget to run build before running tests. But I'd be fine to remove it and keep it separate.